$OpenBSD: patch-hackage-security_cabal,v 1.1 2018/01/22 21:23:53 kili Exp $

Some (not all) stupid "metadata" changes, not contained in the
distribution file.

http://hackage.haskell.org/package/hackage-security-0.5.2.2/revisions/

Index: hackage-security.cabal
--- hackage-security.cabal.orig
+++ hackage-security.cabal
@@ -99,9 +99,9 @@ library
                        base16-bytestring >= 0.1.1   && < 0.2,
                        base64-bytestring >= 1.0     && < 1.1,
                        bytestring        >= 0.9     && < 0.11,
-                       Cabal             >= 1.14    && < 1.26,
+                       Cabal             >= 1.14    && < 2.2,
                        containers        >= 0.4     && < 0.6,
-                       directory         >= 1.1.0.2 && < 1.3,
+                       directory         >= 1.1.0.2 && < 1.4,
                        ed25519           >= 0.0     && < 0.1,
                        filepath          >= 1.2     && < 1.5,
                        mtl               >= 2.2     && < 2.3,
@@ -111,7 +111,7 @@ library
                        -- 0.4.2 introduces TarIndex, 0.4.4 introduces more
                        -- functionality, 0.5.0 changes type of serialise
                        tar               >= 0.5     && < 0.6,
-                       time              >= 1.2     && < 1.7,
+                       time              >= 1.2     && < 1.9,
                        transformers      >= 0.4     && < 0.6,
                        zlib              >= 0.5     && < 0.7,
                        -- whatever versions are bundled with ghc:
@@ -149,9 +149,7 @@ library
                        UndecidableInstances
   -- use the new stage1/cross-compile-friendly Quotes subset of TH for new GHCs
   if impl(ghc >= 8.0)
-    -- place holder until Hackage allows to edit in the new extension token
-    -- other-extensions: TemplateHaskellQuotes
-    other-extensions:
+    other-extensions: TemplateHaskellQuotes
   else
     other-extensions: TemplateHaskell
 
@@ -205,9 +203,7 @@ library
 
   if impl(ghc >= 7.10)
      other-extensions: AllowAmbiguousTypes
---                       StaticPointers
--- ^^^ Temporarily disabled because Hackage doesn't know yet about this
--- extension and will therefore reject this package.
+                       StaticPointers
 
 test-suite TestSuite
   type:                exitcode-stdio-1.0
